bin/specextract

```diff
@@ -1331,7 +1351,7 @@ def spectra(args):
                 readout_streak_spec(specfile=phafile,infile=fullfile,src_x=float(args["skyx"]),src_y=float(args["skyy"]))
 
             try:
-                pardict[key_id].update({"phafile" : phafile})
+                pardict[key_id].update({"phafile" : phafile})                    
             except NameError:
                 pardict = {key_id : {"phafile" : phafile}}
                 
```

Comment:
The following should be changed to

```python
    except OSError as exc:
        raise IOError(f"Failed to extract spectrum for {fullfile}") from exc
```

although I think this should be moved closer to where the error is going to be raised , but that might be too much of a change now.